vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -183,6 +183,7 @@
     <ClInclude Include="Base\CoreNativeModules.h" />
     <ClInclude Include="Base\CxxReactIncludes.h" />
     <ClInclude Include="Base\FollyIncludes.h" />
+    <ClInclude Include="ReactHost\JSCallInvokerScheduler.h" />
```

Comment:
Just checking: Was this just a missing file in the vcxproj or is it still here from an intermediate change?